### PR TITLE
refactor(lightblue) : Convert libs to dotnet standard 2.0

### PR DESCRIPTION
**Pull request by seanswalling**

### CODE REVIEW

File: LightBlue/Hosted/HostedAzureQueueStorageClient.cs [hunk 1]
```diff
@@ -21,9 +21,25 @@ public IAzureQueue GetQueueReference(string queueName)
 
         public IEnumerable<IAzureQueue> ListQueues()
         {
-            return _cloudQueueClient.ListQueues()
-                .Select(q => new HostedAzureQueue(q))
-                .ToArray();
+            //var segment = _cloudQueueClient.ListQueuesSegmentedAsync(null).GetAwaiter().GetResult();
```
Comment (eric-winkler) [inline LightBlue/Hosted/HostedAzureQueueStorageClient.cs]: This method looks unfinished

File: LightBlue/Setup/Contexts/LightBlueLogicalCallContext.cs [hunk 2]
```diff
@@ -1,32 +1,37 @@
...
     class LightBlueLogicalCallContext : LightBlueContextBase
     {
         private const string Key = "LightBlueLogicalCallContext";
-
+        private readonly AsyncLocal<StandaloneConfiguration> _value = new AsyncLocal<StandaloneConfiguration>();
+        
         public bool IsInitialized()
         {
-            return CallContext.LogicalGetData(Key) != null;
+            return _value != null;
```
Comment (NineDelta) [inline LightBlue/Setup/Contexts/LightBlueLogicalCallContext.cs]: This should probably be ```_value.Value != null```

File: LightBlue/Setup/Contexts/LightBlueLogicalCallContext.cs [hunk 3]
```diff
@@ -1,32 +1,37 @@
...
 
         public StandaloneConfiguration Config
         {
             get
             {
-                var value = (StandaloneConfiguration)CallContext.LogicalGetData(Key); 
-                if(value == null) throw new InvalidOperationException("Logical call context has not been initialized for this thread.");
-                return value;
+                //var value = (StandaloneConfiguration)CallContext.LogicalGetData(Key); 
+                if (_value == null) throw new InvalidOperationException("Logical call context has not been initialized for this thread.");
```
Comment (NineDelta) [inline LightBlue/Setup/Contexts/LightBlueLogicalCallContext.cs]: This should probably be ```if (_value.Value == null)```

**Comment by eric-winkler** [comment]
Superseded by https://github.com/LightBlueProject/LightBlue/pull/145

**Pull request closed without merging**